docs/abs_utils.md

```diff
@@ -60,22 +61,16 @@ Run the script from the `gatling-puppet-load-test` directory:
 bundle exec ruby ./util/abs/provision_pe_xl_nodes.rb
```

Comment:
I would change the header from "default options" to "Most Common Execution".  And I think you commonly pass in the ticket number as the id don't you?  if so add that to the execution.  The idea is to get across to the reader the most common way the script it run... like giving them an opinionated way to run it.  The value here is if I read this in 6 months will I know how we usually run it?  From the current reading I might think that "default options" was just showing me the simplest execution, not the opinionated way to run it.  Does that make sense?

Reply:
I think it seems odd to show an example of running it with options before showing an example of running it without. This is why I listed the example with no options specified (i.e. default options) first. 

I've updated the section to add a 'Common usage scenarios' heading and more descriptive text.